packages/expo-location/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 ### 🐛 Bug fixes
 
 - Added some safety checks to prevent `NullPointerExceptions` in background location on Android. ([#8864](https://github.com/expo/expo/pull/8864) by [@mczernek](https://github.com/mczernek))
+- Add isoCountryCode to Address type and reverse lookup. ([#8913](https://github.com/expo/expo/pull/8913) by [@bycedric](https://github.com/bycedric))
```

Comment:
```suggestion
- Add `isoCountryCode` to `Address` type and reverse lookup. ([#8913](https://github.com/expo/expo/pull/8913) by [@bycedric](https://github.com/bycedric))
```

Reply:
[docs] Add code quotes around properties in changelog